README.md

```diff
@@ -20,9 +20,7 @@ See [LICENSE.txt](sycl/LICENSE.TXT) for details.
 See [CONTRIBUTING.md](CONTRIBUTING.md) for details.
 
 ## Sub-projects Documentation
-
-* oneAPI Data Parallel C++ compiler - See
-  [GetStartedGuide.md](sycl/doc/GetStartedGuide.md)
+ * oneAPI Data Parallel C++ compiler - See [DPC++ Documentation](https://intel.github.io/llvm-docs/)
```

Comment:
It's exceeding 80 chars. Please also return empty line below heading.

Reply:
Fixed